Automattic/jetpack
# Launchpad: Add helper function to get the task statuses from site options

## Proposed changes:

* It looks like we will use this line of code frequently given the new method of marking tasks as complete, so create a function to return the task statuses from site options to avoid repetition.
* Update existing instances of the line of code to use the function instead.
* No visual/functional changes.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply this patch to your sandbox and sandbox the API
* Create a new site from `/start` and ensure the Launchpad still works as expected.
* You can double-check that you're getting the expected results from the developer console, too.
